webview-ui/src/index.tsx

```diff
@@ -1,4 +1,9 @@
 import React from "react";
+import "@patternfly/patternfly/patternfly.css";
```

Comment:
The css is already imported below the component imports. I don't think it really matters, just should only import everything once.